### PR TITLE
Update behavior

### PR DESCRIPTION
**Pull request by jjehl**
The 10 ms actual refresh could be unadpated to wifi router. If you ask more quickly than the router can answer it occurs some random delay (need to be tested on more routers).

When you update the rest link with a bad one, you stop the refresher, so it will never restart. Fixed the issue with a getRobot() on update.

And some more independant files and images I use on my robots.

### CODE REVIEW

File: tags/robot.tag [hunk 1]
```diff
@@ -60,12 +60,17 @@
       self._url_open = !self._url_open;
     };
 
-    self.change_url = function(e){
+   self.change_url = function(e){
```
Comment (show0k) [inline tags/robot.tag]: Indentation

File: tags/robot.tag [hunk 2]
```diff
@@ -60,12 +60,17 @@
...
       if (e.keyCode == 13) {
+	    Poppy.url = document.getElementById('poppy_url').value;
+		sessionStorage.setItem("poppy_url",Poppy.url);
```
Comment (show0k) [inline tags/robot.tag]: Lines 66, 67, 71 and 72 have wrong indentations

**Review by show0k — CHANGES_REQUESTED** [review]
Some lines have a broken indentation

**Comment by jjehl** [comment]
Indentation should be fine now. Finally, I prefer to add a refresher on error, better to call a GetRobot because it avoid to have multiple GetRobot.

**Review by show0k — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**